### PR TITLE
tizenrt: Use TizenRT version to 2.0 update defconfigs

### PR DESCRIPTION
**Pull request by rzr**
Align to TizenRT's default config, but disable SSS for ARTIK053
also preserve debug options for debug configuration.

Because current version of TizenRT in IoT.js docker image is outdated,
let's fetch tag from upstream at build time,
to speed up build time, this part can be eventually removed
if image is updated with latest TizenRT's sources.

jerryx extra lib has been dropped for now,
this should be reintroduced later (once TizenRT upgraded IoT.js ?)

Bug: https://github.com/Samsung/iotjs/issues/1777
Forwarded: https://github.com/Samsung/iotjs/pull/1780
IoT.js-DCO-1.0-Signed-off-by: Philippe Coval p.coval@samsung.com

### CODE REVIEW

File: tools/travis_script.py [hunk 1]
```diff
@@ -151,10 +152,8 @@ def build_iotjs(buildtype, args=[], env=[]):
             exec_docker(DOCKER_TIZENRT_OS_PATH, [
                         'make', 'IOTJS_ROOT_DIR=' + DOCKER_IOTJS_PATH,
                         'IOTJS_BUILD_OPTION='
-                        '--profile=test/profiles/tizenrt.profile',
```
Comment (daeyeon) [inline tools/travis_script.py]: Any reason we should drop jerry-ext? It seems better to follow the above FIXME.

Comment (LaszloLango) [reply tools/travis_script.py]: I think it can be deleted because the `jerry-debugger` is not enabled by default, so `jerry-ext` is not needed after #1798

Comment (LaszloLango) [reply tools/travis_script.py]: (If we keep this change, then the comment must be deleted as well, because it is not valid after this patch)

**Comment by rzr** [comment]
Yes this part was removed because lib is no more built:

https://travis-ci.org/Samsung/iotjs/jobs/4596535933#

arm-none-eabi-ld: cannot find -ljerry-ext


I am updating patch and remove the header comment as suggested

**Pull request merged**